README.md

```diff
@@ -98,6 +98,13 @@ Experiment with flagd in your browser using [the Killercoda tutorial](https://ki
       -d '{"flagKey":"myStringFlag","context":{}}' -H "Content-Type: application/json"
     ```
 
+    For Windows use bash or use the following:
```

Comment:
```suggestion
    For Windows we recommend using a [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) terminal. 
    Otherwise, use the following with `cmd` or `powershell` 
```

Reply:
powershell wont work with this curl, but I will commit the other suggestions 👍